Rakefile

```diff
@@ -61,62 +61,67 @@ namespace 'gem' do
   desc 'Build a binary gem'
   task :binary, :ruby2_32, :ruby2_64, :ruby21, :ruby21_64, :ruby22, :ruby22_64, :ruby23_32, :ruby23_64, :ruby24_32, :ruby24_64 do |task, args|
     # These are just what's on my system at the moment. Adjust as needed.
+    # ri refers to RubyInstaller, ruby 2.3 and prev were built with RubyInstaller (:ri),
+    # 2.4 and later with RubyInstaller2 (:ri2)
+    # pre variable allows local builds with ruby & DevKit not at drive root
+
+    dk_path = {}
+    if ENV['APPVEYOR'] =~ /true/i
+      pre = "C:"
+      dk_path[:ri]    = "C:/ruby23/DevKit/bin;C:/ruby23/DevKit/mingw/bin"
+      dk_path[:ri_64] = "C:/ruby23-x64/DevKit/bin;C:/ruby23-x64/DevKit/mingw/bin"
+    else
+      ENV['BASEPATH'] = ENV['PATH'].gsub(/\\/, '/')
+      pre = "C:"
```

Comment:
How about use `pre = ENV["LOCAL_DEVKIT_PATH"] || "C:"` here?
This code introduces build environment to be configurable.
This is for future work.
You don't have to fix it.

Reply:
> How about use pre = ENV["LOCAL_DEVKIT_PATH"] || "C:" here?

I believe `ENV["LOCAL_DEVKIT_PATH"]` is specific to RubyInstaller builds, and they have been replaced with RubyInstaller2 (MSYS2) builds.

The only reason I added the `pre` variable is because I wanted to check `bundle exec rake gem:binary` locally.  My normal day-to-day ruby is at `C:\`, but everything else is at `E:\r_builds\`.

It seemed like an easy way to allow myself (and others) to check a local build without heavy editing to `Rakefile` (if all they needed a was different base path)...

Reply:
I see. let’s leave as-is.